tools/bin/check-duplicated-storages.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+backup_storage_output=$(kubectl get backupstorage -n everest-system)
+backup_storage_names=($(echo "$backup_storage_output" | awk 'NR>1 {print $1}'))
```

Comment:
I think you can get this done in a single command but please test it, I haven't :sweat_smile: 
```suggestion
backup_storage_names=$(kubectl get backupstorage -n everest-system --no-headers -o=custom-columns=NAME:.metadata.name)
```

Reply:
Checked, it doesn't work bc the output is one line instead of an array so the script can't iterate. However I've found another way to combine the two commands into one, thanks for the idea!